.prettierrc.js

```diff
@@ -4,4 +4,6 @@ module.exports = {
   bracketSpacing: false,
   singleQuote: true,
   trailingComma: 'all',
+  semi: true,
+  endOfLine: 'lf',
```

Comment:
There is no need to worry about this in unix systems, it is default value. On other systems, [git should manage that](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings)